CHANGELOG.md

```diff
@@ -84,6 +84,18 @@ systemctl restart docker` to reload changes and (re)start the docker daemon.
 - Fix an issue where a volume mount could inadvertently create a bind mount [#25309](https://github.com/docker/docker/pull/25309)
 - `docker service create --mount type=bind,...` now correctly validates if the source path exists, instead of creating it [#25494](https://github.com/docker/docker/pull/25494)
 
+### Swarm
```

Comment:
@aaronlehmann can you please move this above Volume section? trying to make it alphabetical, although it's quite inconsistent through releases.